doc/realtime_servo/realtime_servo_tutorial.rst

```diff
@@ -17,7 +17,10 @@ Getting Started
 ---------------
 This tutorial demonstrates the servo node with a UR5 Gazebo simulation. If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
 
-Clone `universal_robot melodic-devel branch <https://github.com/ros-industrial/universal_robot.git>`_ into the same catkin workspace from `Getting Started`: ::
+Clone `universal_robot melodic-devel branch <https://github.com/ros-industrial/universal_robot.git>`_ into the same catkin workspace from `Getting Started`:
+
+**NOTE: Please stick to older version of Universal Robot Github (1.2.7 ) as newer versions (1.3.x) are incompatible with the servo tutorial.** (`Src <https://github.com/ros-planning/moveit_tutorials/commit/f9fba77cd6cab31c9ae360bbaca3896a6acef446>`_)
```

Comment:
I'd recommend deleting this link. I don't think it adds anything, really, and certainly needs a better description than "Src". Looks good to me besides that. Thanks!

```suggestion
**NOTE: Please stick to older version of Universal Robot Github (1.2.7 ) as newer versions (1.3.x) are incompatible with the servo tutorial.**
```